spec/takeaway_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'takeaway'
+
+describe Takeaway do
+  subject(:takeaway) { described_class.new(menu) }
+  let(:menu) {double :menu}
+  let(:dish) {double :dish}
+  let(:dish2) {double :dish2}
+
+  it { is_expected.to respond_to(:select).with(2).arguments }
+
+  it 'selects items from the menu which is stored as an order' do
+    expect(takeaway.select(:salad, 2)).to eq "You have added 2 salad(s) to your basket"
```

Comment:
Metrics/LineLength: Line is too long. [87/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)